### PR TITLE
Initialized a migration and seeded dummy data into member table

### PR DESCRIPTION
**Pull request by Aruaan**
Dropped the initial database and created a migration, after which a custom seeder was made to simulate adding data into one table.

### CODE REVIEW

File: src/database/database.module.ts [hunk 1]
```diff
@@ -8,9 +14,11 @@ import {TypeOrmModule} from '@nestjs/typeorm'
       port: 3306,
       username: 'root',
       password: 'samojako',
-      database: 'gym_db',
-      entities: ["dist/**/*.entity{.ts,.js}"],
```
Comment (SreckoStojicNS) [inline src/database/database.module.ts:12]: Mogao si i ovako, samo si trebao da nazivas fajlove sa .entity.ts.
meal.entity.ts
member.entity.ts i tako dalje

File: src/database/entities/Equipment.ts [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+  @Column ({length: 30, type:"varchar"})
   type:string
 
-  @Column ()
+  @Column ({type:"date", nullable:true})
   purchase_date: Date
 
-  @Column ({length: 255})
+  @Column ({length: 255, type:"varchar", nullable:true})
   notes: string
```
Comment (SreckoStojicNS) [inline src/database/entities/Equipment.ts:19]: Ako ti je nullable true onda moras i notes da stavis pravi tip.
notes: string | null

File: src/database/entities/Equipment.ts [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+  @Column({length: 30, type:"varchar"})
   name: string
 
-  @Column ({length: 30})
+  @Column ({length: 30, type:"varchar"})
   type:string
 
-  @Column ()
+  @Column ({type:"date", nullable:true})
   purchase_date: Date
```
Comment (SreckoStojicNS) [inline src/database/entities/Equipment.ts:16]: I ovde ti nije dobar tip ako je nullable

File: src/database/entities/Measurement.ts [hunk 4]
```diff
@@ -1,21 +1,24 @@
 import { Column, Decimal128, Entity, JoinColumn, ManyToMany, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
 import { Member } from "./Member";
 
-Entity()
+@Entity()
```
Comment (SreckoStojicNS) [inline src/database/entities/Measurement.ts:4]: mozes proslediti name kako da ti se zove tabela u bazi sa { name: ' '}

File: typeOrm.config.ts [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import { DataSource } from "typeorm";
+
+export default new DataSource({
+  type: 'mysql',
+  host: '127.0.0.1',
+  port: 3306,
+  username: 'root',
+  password: 'samojako',
```
Comment (VlatkovicB) [inline typeOrm.config.ts:8]: Istrazi environment varijable, ovakve stvari ne bi trebale da se nalaze u kodu.

**Review by SreckoStojicNS — APPROVED** [review]

**Pull request merged**